packages/mdc-toolbar/_mixins.scss

```diff
@@ -25,17 +25,19 @@
   display: flex;
   align-items: center;
   justify-content: center;
-
-  // stylelint-disable at-rule-empty-line-before, block-closing-brace-newline-after
-  @if $padding-type == menu {
-    padding: $mdc-toobar-element-vertical-padding;
-  } @else {
-    padding: $mdc-toobar-element-vertical-padding 8px;
-  }
   // stylelint enable
 
   border: none;
   background-color: inherit;
   color: inherit;
   text-decoration: none;
+
+  // stylelint-disable at-rule-empty-line-before, block-closing-brace-newline-after
+  @if $padding-type == menu {
+    padding: $mdc-toobar-element-vertical-padding;
+  }
```

Comment:
It looks like you covered the existing ones, but should we un-space this new if/else too?